UserCard.jsx

```diff
@@ -0,0 +1,39 @@
+function UserCard({ userId }){
+  const [user, setUsers] = React.useState([]);
+  const [isLoading, setIsLoading] = React.useState(true)
+  
+  React.useEffect(()=>{
+    getUser(userId)  
+      .then(user =>{
+        setUsers(user);
+        setIsLoading(false);
+      });
+  },[]);
```

Comment:
This works as is, but generally it's a good idea to pass any properties that the`useEffect` function depends on in this dependency array. If this component ever got a new `userId` for some reason, we'd want to re-run this function to do the re-fetching. A good rule of thumb is that if you're accessing either a prop or a piece of state that gets updated outside of the `useEffect` callback, you want to include it in the dependency array.

Reply:
```suggestion
  },[userId]);
```